polaris/polaris/migrations/0014_auto_20220207_0121.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.2.9 on 2022-02-07 01:21
+
+from django.db import migrations, models
+import polaris.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("polaris", "0013_auto_20211011_1956"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="PolarisHeartbeat",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("key", models.CharField(max_length=80, unique=True)),
```

Comment:
80 is arbitrary but since this is a key i think we should limit the number of characters. doesn't make sense to have a key thats 1000 characters long (if we use TextField)